README.md

```diff
@@ -5,7 +5,7 @@ A set of lighting fixture definitions usable with PIXILAB Blocks 2 or later.
 
 To use these files, copy the desired brand name folder(s) into the fixtures directory located inside your PIXILAN-root directory.
 
-**NOTE:** These definitions were partially derived from the excellent _Open Fixtures Library_ found at https://open-fixture-library.herokuapp.com
+**NOTE:** These definitions were partially derived from the excellent _Open Fixture Library_ found at https://open-fixture-library.org ([GitHub repository](https://github.com/OpenLightingProject/open-fixture-library))
```

Comment:
@TheWizz I just saw your [Wiki page](https://int.pixilab.se/docs/blocks/lighting/fixture_definitions); could you please make these changes there, too?

Reply:
Done